app/api/referral/invitations/route.ts

```
@@ -0,0 +1,165 @@
+    .eq('owner_id', user.id)
+
+  const ownedIds = (companies ?? []).map((c) => c.id)
+  if (ownedIds.length === 0) {
+    return NextResponse.json({ data: [], total: 0, page, pageSize })
+  }
+
+  if (companyId && !ownedIds.includes(companyId)) {
+    return NextResponse.json({ error: 'Invalid company filter' }, { status: 403 })
+  }
+
+  let query = supabase
```

ESLint

[error] 'query' is never reassigned. Use 'const' instead.

(prefer-const)